modules/core/form/js/form_protection.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file
+ * Show warning when a user is about to navigate away from an unsaved, dirty form.
+ */
+(function ($, Drupal) {
+
+    "use strict";
+
+    Drupal.behaviors.form_protection = {
+
+        initialFormState: {},

```

Comment:
We can move `initialFormState` up to the scope of the [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) and avoid the need for the `behavior.initialFormState` references.

```suggestion

    const initialFormState = {};

    Drupal.behaviors.form_protection = {

```